source/AI.cpp

```diff
@@ -3224,7 +3330,9 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			ship.SetTargetStellar(system->FindStellar(bestDestination));
 	}
 
-	if(activeCommands.Has(Command::NEAREST))
+	if(activeCommands.Has(Command::NEAREST_ASTEROID))
+		PlayerTargetAsteroid(ship, player, activeCommands);
+	else if(activeCommands.Has(Command::NEAREST))
```

Comment:
Must this new command come first?

Reply:
In practice, yes I think so.  I am trying to preserve the old behavior.

So I am adding a `V` shortcut to target asteroids specifically.  However, currently in the game pressing `R` when there is no hostiles will also target an asteroid.  Players (myself included) who have a lot of mining experience in-game have formed a habit of using `R` for targeting asteroids so I wish to preserve it.

If hostiles are in the system, ships will continue to be targeted by default.

Reply:
My question is, is there a reason to handle `NEAREST_ASTEROID` before `NEAREST`? Or can you handle the more commonly used option (nearest) before the niche command?

Reply:
I don't see a reason why not looking at the code.  I'll move it down and play test it.  If it works as intended I'll push.  If not I'll share something I learn from it.